servant-server/src/Servant/Server/Internal.hs

```diff
@@ -531,7 +531,7 @@ instance (KnownSymbol sym, HasServer api context)
 
 -- | If you use @'QueryParamForm' "formName" Text@ in one of the endpoints for your API,
 -- this automatically requires your server-side handler to be a function
--- that takes an argument of type @'Maybe' BookSearchParams'@.
+-- that takes an argument of type @'Maybe Either BookSearchParams'@.
```

Comment:
Typo?

Reply:
hmm. I'm not too happy with this documentation here. I'll try to improve it.